app/scripts/map/cartodb-vis-directive.js

```diff
@@ -147,6 +192,59 @@
                 $('div.cartodb-tooltip .tooltip-tracts').parent().css('display', 'block');
             }
         }
+
+        function onPointsLayerClicked(event, latLng, pos, data) {
+            var popupScope = $scope.$new(true);
+            popupScope.loading = true;
+            makePopupAtLatLng(popupScope, latLng);
+            var client = new cartodb.SQL({user: Config.cartodb.account});
+            var sql = 'SELECT {{ fields }} FROM {{table}} ' +
+                'WHERE ST_Within(' +
+                    'ST_Transform(the_geom, 4326), ' +
+                    'ST_Buffer(ST_SetSRID(ST_MakePoint({{ x }}, {{ y }}),4326), {{ tolerance }})' +
+                ')';
+            client.execute(sql, {
+                table: Config.cartodb.tableName,
+                x: data.longitude,
+                y: data.latitude,
+                fields: ['ein_new', 'organization_new'].join(','),
+                tolerance: 0.0001
+            }).done(function (data) {
+                popupScope.rows = data.rows;
+                popupScope.loading = false;
+                makePopupAtLatLng(popupScope, latLng);
+            });
+        }
+
+        function onPointsLayerFeatureOver(event, latLng, pos, data) {
+            var popoverScope = $scope.$new(true);
+            /* jshint camelcase:false */
+            popoverScope.orgName = data.organization_new;
+            /* jshint camelcase:true */
+            popoverScope.resultCount = OrgCountService.atLatLng(data.latitude, data.longitude);
+            var popoverHtml = $compile(orgPopoverTemplate)(popoverScope);
+            popoverScope.$apply();
+            $popover.html(popoverHtml[0]);
+            $popover.css({
+                top: pos.y + 5,
+                left: pos.x + 5
+            });
+            $popover.addClass('visible');
+        }
+
+        function onPointsLayerFeatureOut() {
+            $popover.removeClass('visible');
+        }
+
+        function makePopupAtLatLng(scope, latLng) {
+            var popupHtml = $compile(orgPopupTemplate)(scope);
+            scope.$apply();
+            L.popup({maxWidth: 400})
```

Comment:
Does this style have to be hard-coded here?

Reply:
IIRC yes, since it forces the popup width making the map pan correctly. 

@designmatty if you need to tweak the popup styling and it works out that this can be moved to CSS, feel free to do so.